packages/server-admin-ui/src/views/ServerConfig/BasicProvider.js

```diff
@@ -655,6 +686,12 @@ const NMEA0183 = props => {
         value={props.value.options}
         onChange={props.onChange}
       />
```

Comment:
If the input is from a multiplexer and contains data both with and without checksums we can not validate the checksums and generate missing checksums. I suppose that is ok.

I would rather show the option always but in disabled state, not hide it, so that the user gets a hint that it is there.